src/person.rb

```diff
@@ -1,21 +1,25 @@
-require_relative "Nameable.rb"
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'nameable'
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :parent_permission, :rentals
 
-  def initialize(name = "Unknown", age = 0, parent_permission = true)
+  def initialize(name: 'Unknown', age: 0, parent_permission: true)
+    super()
     @id = generate_id
-    @name = name
     @age = age
+    @name = name
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ] Just as I mentioned in my previous comment, a similar implementation should be present in this `Person class` as well :+1: 
- [ ]  Per the project requirements, you needed to:

> Create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental. The following should be implemented:
> Create the has-many side of Person and Rental (a person has many rentals).

- Currently, this isn't being implemented in the `Person class`. I will suggest you include a method for adding rentals in the class. The person itself should be sent as a parameter to create the new rental thereby achieving the `has-many` association. :wink: 
- Here is a [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship%20def%20add_rental(person,%20date)) to an example of this implementation in a previous lesson